clientd3d/object.h

```diff
@@ -79,6 +79,7 @@ typedef struct {
 		                     this field gives amount of object */
    DWORD     temp_amount;         /* Scratch field used when user is selecting amount of object */
    int       flags;               /* Flags describing various properties of objects */
+   int       rarity;              /* Used to draw object text differently based on rarity tiers */
```

Comment:
Type should be item_rarity_grade?

Reply:
Yes, same as the previous instance. I have changed it accordingly.